tests/fixtures/simple2_yaml.cfg

```diff
@@ -0,0 +1,8 @@
+scheduler:
+    jobname:
+        - extra_stuff: 12345
+        - account: user_account
```

Comment:
This is interesting....I wasn't sure exactly what pyyaml would do with this structure, but it seems it loads as a list of dicts. Does that count as a new level of depth in the depth algorithm?